tools/build-release-tarball

```diff
@@ -114,9 +114,11 @@ rm -f zproject/dev-secrets.conf
 # We don't need duplicate copies of emoji with hashed paths, and they would break bugdown
 find prod-static/serve/generated/emoji/images/emoji/ -regex '.*\.[0-9a-f]+\.png' -delete
 
+./scripts/lib/create-production-venv --no-apt $prefix
+
 cd "$TMPDIR"
 
-tar --append -f "$TARBALL" "$prefix/prod-static" "$prefix/build_id" "$prefix/version" "$prefix/staticfiles.json" "$prefix/webpack-stats-production.json"
+tar --append -f "$TARBALL" "$prefix/prod-static" "$prefix/build_id" "$prefix/version" "$prefix/staticfiles.json" "$prefix/webpack-stats-production.json" "$prefix/zulip-py3-venv"
```

Comment:
I think we want to be including the source tarballs as one would download from `pypi`, not the built/installed copies in the virtualenv; the latter won't relocate properly between different architectures... so I think we want to run an explicit `pip download` to get the packages...

Reply:
OK.

Reply:
> the latter won't relocate properly between different architectures

Alternatively, the build can be made on multiple architectures and distribution.

Reply:
Such as zulip-server-1.6.1-3.4-xenial-x64.tar.gz.

Reply:
Another example, https://bitcoin.org/bin/bitcoin-core-0.15.0.1/, with builds for:
- aarch64-linux-gnu
- arm-linux-gnueabihf
- i686-pc-linux-gnu
- osx
- osx64
- win32-setup
- win32
- win64-setup
- wint64
- x86_64-linux-gnu
- generic
- torrent

Reply:
@rht while that is possible, it's completely unnecessary for the `pip` packages.  We can just ship the results of `pip download` and that will be good enough.